music21/expressions.py

```diff
@@ -517,21 +519,205 @@ def fillListOfRealizedNotes(
         fillObjects.append(firstNote)
         fillObjects.append(secondNote)
 
+    def resolveOtherPitches(self, srcPitch: pitch.Pitch):
```

Comment:
needs public docs -- perhaps give the demo here of what it does with those types.  Not a fan of the name "other pitches" -- doesn't tell me anything about what they are.

Reply:
Nope, can't do that exactly. `Ornament.updateAccidentalDisplay` doesn't have the necessary params in hand to pass to such a method.  It needs to be able to get and modify the ornamental pitches directly, assuming someone else has set them up.  So I will still need `Ornament.resolveOrnamentalPitches(src and maybe keysig)` as well as `Turn.upperOrnamentalPitch`,` Turn.lowerOrnamentalPitch`, and `Mordent/Trill.ornamentalPitch`, and maybe `Ornament.ornamentalPitches`, too, we'll see.

Reply:
Done.